src/services/ieq/ieq.js

```diff
@@ -284,7 +284,11 @@ IEQ.prototype._updateData = function(callback, dataConfig, funcName){
 		}
 	}, function(dnId, err, data){
 		if(err) {
-			Logger.error("["+that.dataConfig.sensors+"] Recv err: "+JSON.stringify(err));
+			if (that.dataConfig && that.dataConfig.sensors) Logger.error("["+that.dataConfig.sensors+"] Recv err: "+JSON.stringify(err));
```

Comment:
I think this check could be simplified by ensuring that that.dataConfig exists.
I assume, the error should be displayed, otherwise this line is useless.